app/views/personas/index.html.erb

```diff
@@ -45,6 +46,7 @@
             row.with_cell { govuk_boolean_tag(staff.reverse_decision_permission) }
             row.with_cell { govuk_boolean_tag(staff.support_console_permission) }
             row.with_cell { govuk_boolean_tag(staff.withdraw_permission) }
+            row.with_cell { govuk_boolean_tag(staff.withdraw_permission) }
```

Comment:
```suggestion
            row.with_cell { govuk_boolean_tag(staff.verification_permission) }
```

Could we also put this before the `withdraw_permission` to maintain the alphabetical ordering?

Reply:
Done =)